grails-app/services/org/pih/warehouse/requisition/RequisitionService.groovy

```diff
@@ -843,6 +841,5 @@ class RequisitionService {
                 requisition.approvalRequired = false
                 requisition.save()
             }
-        }
```

Comment:
it's weird for me that you and @drodzewicz did the same, but in his PR the change seems different than in yours :thinking: